Lib/fontTools/cffLib/specializer.py

```diff
@@ -50,7 +50,10 @@ def programToCommands(program):
 				try:
 					width = stack.pop(0)
 				except IndexError:
-					raise ValueError(stack)
+					if ignoreErrors:
+						width = ''
```

Comment:
@miguelsousa you sure an empty string `''` for the width is ok here? I thought the args list should be numbers only, but maybe i'm missing something else